packages/common/src/types/collection.ts

```diff
@@ -76,7 +83,9 @@ export type TCollectionRequest = TDatabaseRequest &
  */
 export type TCollectionCreateRequest = TCollectionRequest &
   Pick<TCollection, 'schema'> &
-  Pick<OwnershipAndPermission, 'permission' | 'group'>;
+  Pick<OwnershipAndPermission, 'permission' | 'group'> & {
```

Comment:
I think it should be `groupName` not `group` to keep the consistency among endpoint.

Reply:
If we did, I think we need to change quite a lot, If you want I can make another PR for this naming

I think it consist with PERMISSION as well

```ts
export const OWNERSHIP_TYPE = ['owner', 'group'] as const;

export const PERMISSION_TYPE = ['owner', 'group', 'other'] as const;
```

![image](https://github.com/user-attachments/assets/5e1ee870-a3cd-4a4a-ab84-3661143fb2bb)